taxi/urls.py

```diff
@@ -37,14 +42,35 @@
         ManufacturerDeleteView.as_view(),
         name="manufacturer-delete",
     ),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("cars/create/", CarCreateView.as_view(), name="car-create"),
-    path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
-    path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("cars/", CarListView.as_view(),
+         name="car-list"),
```

Comment:
The same issue here, you should use `-` to split words in URL identification parameter `name`, not the `_`. So, `car-list` should be `car_list`.